modules/yieldlabBidAdapter.js

```diff
@@ -24,19 +24,32 @@ export const spec = {
    * @param validBidRequests
    * @returns {{method: string, url: string}}
    */
-  buildRequests: function (validBidRequests) {
+  buildRequests: function (validBidRequests, bidderRequest) {
     const adslotIds = []
     const timestamp = Date.now()
+    const query = {
+      ts: timestamp,
+      json: true
+    }
 
     utils._each(validBidRequests, function (bid) {
       adslotIds.push(bid.params.adslotId)
+      if (bid.params.targeting) {
+        query.t = createQueryString(bid.params.targeting)
+      }
     })
 
+    if (bidderRequest && bidderRequest.gdprConsent) {
+      query.consent = bidderRequest.gdprConsent.consentString
+      query.gdpr = (typeof bidderRequest.gdprConsent.gdprApplies === 'boolean') ? bidderRequest.gdprConsent.gdprApplies : true
```

Comment:
just verifying that you want to default `gdprApplies` to `true`

Reply:
Yes, gdprApplies=true should be the default. Thanks for making sure!